API-strategie-extensies/ext-geospatial.md

```diff
@@ -135,23 +135,29 @@ The guiding principles for CRS support:
 - Exchange format (notation) ETRS89 and WGS84 X Y in decimal degrees: DD.ddddddddd (for example: `5.962376256, 52.255023450`)
 - Exchange format (notation) RD and Web Mercator X Y in meters: `195427.5200 311611.8400`
 
-<div class="rule" id="api-40">
-  <p class="rulelab"><strong>API-40</strong>: Pass the coordinate reference system (CRS) of the request as a parameter</p>
+<div class="rule" id="api-41">
+  <p class="rulelab"><strong>API-41</strong>: Pass the coordinate reference system (CRS) of a request parameter as a parameter</p>
```

Comment:
Pass the coordinate reference system (CRS) of the geometry in a request parameter as a parameter.